### PR TITLE
Allow changing both vehicle colors

### LINKED ISSUE #24: Only one car color is handled by /color

### PR DESCRIPTION
**Pull request by dracc**
Closes #24 
Requires #23

### CODE REVIEW

File: scripts/module/color.nut [hunk 1]
```diff
@@ -6,12 +6,23 @@ function onPlayerCommand( player, cmd, text ) {
...
+          local secondArg = GetTok(text, " ", 2);
+          if (IsNum(secondArg)) {
+            v.Colour2 = secondArg.tointeger();
+          }
+        }
+        return 1;
+      }
+    }
+    MessagePlayer( "[#ff0000]Usage: /" + cmd + " <Colour1ID>, /" + cmd +
+                   " <Coulour1ID> <Colour2ID>", player );
```
Comment (JayFoxRox) [inline scripts/module/color.nut]: If only one colour is provided, it's probably a better idea to set both, color 1 and 2 to the same one.

It's what I most often want to do tbh.

**Comment by dracc** [comment]
I just noticed that the `GetTok()` function is not a squirrel feature but something defined in `admin.nut`.

**Comment by JayFoxRox** [comment]
Yes, I was aware and would have preferred `split()` once, and then using the array elements - also makes the code easier to read, probably. However, I then googled to make sure it wasn't a vc-mp function and found http://liberty-unleashed.co.uk/LUWiki/Squirrel/Server/Functions/Utils/GetTok , not realizing it isn't a vc-mp function (as last part of URL suggests), but only Liberty Unleashed.

I'm fine with `GetTok` temporarily, but ideally we'd fix it asap. Up to you.

**Comment by dracc** [comment]
> I'm fine with `GetTok` temporarily, but ideally we'd fix it asap. Up to you.
To clarify, current implementation is broken. I should change it into using `split()`.

**Comment by dracc** [comment]
Updated and tested, works as expected.

**Comment by dracc** [comment]
Rebased and ready for merge.

File: scripts/module/color.nut [hunk 2]
```diff
@@ -6,12 +6,22 @@ function onPlayerCommand( player, cmd, text ) {
...
+        v.Colour1 = col2;
+        if (args.len() >= 2 && IsNum(args[1])) {
+          col2 = args[1].tointeger();
+        }
+        v.Colour2 = col2;
+        return 1;
+      }
+    }
+    MessagePlayer( "[#ff0000]Usage: /" + cmd + " <Colour1ID>, /" + cmd +
+                   " <Coulour1ID> <Colour2ID>", player );
```
Comment (JayFoxRox) [inline scripts/module/color.nut]: typo: Coulour

File: scripts/module/color.nut [hunk 3]
```diff
@@ -6,12 +6,22 @@ function onPlayerCommand( player, cmd, text ) {
...
+        local col2 = args[0].tointeger();
+        v.Colour1 = col2;
+        if (args.len() >= 2 && IsNum(args[1])) {
+          col2 = args[1].tointeger();
+        }
+        v.Colour2 = col2;
+        return 1;
+      }
+    }
+    MessagePlayer( "[#ff0000]Usage: /" + cmd + " <Colour1ID>, /" + cmd +
```
Comment (JayFoxRox) [inline scripts/module/color.nut]: I'd just prefer `Colour-ID` here. Or in general: `Color-ID` [US]

**Comment by dracc** [comment]
Ready for merge.

**Pull request merged**